.github/workflows/eamxx_default_files.yml

```diff
@@ -0,0 +1,72 @@
+name: inputdata
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron: '00 00 * * *'
+  workflow_dispatch:
+
+jobs:
+  scream-defaults:
+    runs-on: ubuntu-latest
+    outputs:
+      event_name: ${{ github.event_name }}
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@v4
+        with:
+          show-progress: false
+          submodules: false
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v4.7.0
+        with:
+          python-version: "3.11"
+      - name: Run unit tests
+        working-directory: components/eamxx/cime_config/
+        run: |
+          python -m unittest tests/eamxx_default_files.py -v
+
+  notify-scream-defaults:
+    needs: scream-defaults
+    if: failure() && needs.scream-defaults.outputs.event_name == 'pull_request'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create issue
+        run: |
+          if [[ $CLOSE_PREVIOUS == true ]]; then
+            previous_issue_number=$(gh issue list \
+              --label "$LABELS" \
```

Comment:
We should probably add a new label, that is only used by this issue, so that we avoid finding an issue with the same labels that is however not the one that was generated by this action...